doc/bash/symfony_aliases.md

```diff
@@ -40,4 +40,20 @@
 - **sfai**: `php app/console assets:install web --symlink`
 
 ### Symfony server ###
-- **sfs**: `php app/console se:ru`
+- **sfsr**: `php app/console server:run`
+- **sfss**: `php app/console server:status`
+- **sfsq**: `php app/console server:stop`
+
+### Doctrine ###
+docdbcreate=`doctrine:database:create`
```

Comment:
And shouldn't this line be like this? (alias name prefixed with `sf` and the actual alias prefixed with `php app/console`)
```
sfdocdbcreate=`php app/console doctrine:database:create`
```

And the same for all the other aliases from this group.